### PR TITLE
Portfolio Initial Draft

### PR DESCRIPTION
**Pull request by Mmendero**
Editted HTML, CSS, and JS files to give a basic template for portfolio

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,46 @@
...
+      <h1 id="name">Matthew Mendero</h1>
+      <p id="greeting-text"> Welcome to my page!</p>
+    </div>
+
+    <div id="whitespace"> </div>
+
+    <div id="content">
+      <h2 id="name">About Me</h2>
+        <div class="flexbox">
+            <p id="maintext"> Hi I'm Matt! I'm from Garfield, New Jersey. Currently, I am a Sophomore at Montclair State University studying Computer Science. I'm half Filippino from my mother's side and half Haitian from my father's side. My Interest for technology began with my passion for gaming, one thing led to another and I found myself coding for fun. Click the button to scroll through some things that interest me!</p>
```
Comment (wangmengkate) [inline portfolio/src/main/webapp/index.html]: This line is too long. Please try to format the code file and keep each line less than 80 characters.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,46 @@
...
+        </div>
+        <ul class="social-icons">
+            <li><a href="https://github.com/Mmendero" target="_blank"><img src='/images/Github.png' /></a></li>
+            <li><a href="https://twitter.com/m1ett" target="_blank"><img src='/images/Twitter.png' /></a></li>
+            <li><a href="https://www.linkedin.com/in/matthew-mendero-atis-8a1b9314b/" target="_blank"><img src='/images/Linkedin.png' /></a></li>
+        </ul>
+    </footer>
   </body>
+
+
```
Comment (wangmengkate) [inline portfolio/src/main/webapp/index.html]: Please delete the extra empty line. One empty line at the end of a file is good enough

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,89 @@
...
+
+.social-icons li a {
+  border-bottom: none;
+  }
+
+.social-icons li img {
+  width:50px;
+  height:50px;
+  margin-right: 20px;
+}
```
Comment (wangmengkate) [inline portfolio/src/main/webapp/style.css]: Please add a new line at the end of this file.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (Mmendero) [reply portfolio/src/main/webapp/index.html]: Done

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (Mmendero) [reply portfolio/src/main/webapp/style.css]: Done

**Pull request merged**